### ISSUE TITLE
Rebar Lengths 

### ISSUE THREAD

**Issue by khatriharsh** [paireks/T-Rex#4]
As in day to day uses Rebar have specific lenght of 12m after which lap length is provided 
Please incorporate this option

**Comment by paireks** [comment]
You're right with that, do you have some ideas what exactly should be added in code or Grasshopper components?

**Comment by paireks** [comment]
Thank you! :) I'd have to think about some features that can be added there related to this length. If you have some examples how it should look like: feel free to post it there, it will be helpful.
